README.md

```diff
@@ -46,18 +46,15 @@ Design is based on: https://www.figma.com/file/dt1Z45BJkDGOg63IdNYwbR
 
 ### Development
 
-It's best to use local network. Local network is configured using `docker-compose.yml`.
-By typing `docker-compose up` in terminal, you will get:
-- geth eth1 node with private POA network
-- deposit contract deployed on that private eth1 network
-    - contract is always deployed on address `0x2F1598e74b146F5687174C13f8EDCF490B2492e3`
-- lighthouse eth2 private network
-
-Local testnet is started with 16 validators but only 15 are active (run using lighthouse docker validator).
-Url of local testnet beacon node is: `http://localhost:5052`;
-You can use this account to validate using ChainGuardian (skip withdrawal key):
-- private key: `0x03cffafa1cbaa7e585eaee07a9d35ae57f6dfe19a9ea53af9c37e9f3dfac617c`
-- public key: `0x872c61b4a7f8510ec809e5b023f5fdda2105d024c470ddbbeca4bc74e8280af0d178d749853e8f6a841083ac1b4db98f`
+It's best to use local network. Local network is configured using `docker-compose` (currently available lighthouse-testnet.yml).
+For the first time you need to run setup script (`setup.sh`) on path `./testnet/lighthouse`
```

Comment:
Since testnet dir is committed, just put this in small paragraph title "Generating new local testnet config" or something

Reply:
I din't commit validator keystores and configs that why need to be generated

Reply:
commit them, we won't be generating that